src/App.js

```diff
@@ -1,9 +1,16 @@
+import { Route, Routes } from 'react-router-dom';
 import './App.css';
+import BookList from './components/BookList';
+import Navbar from './components/Navbar';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header" />
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<BookList />} />
+        <Route path="/categories" element={<h3>Coming soon</h3>} />
```

Comment:
- [ ] You have done an impressive job with implementing the routes on your project :+1:. However, to fully meet the project requirements, you are required to create a component called `<Categories>` instead of using the `<h3>Coming soon</h3>` element. This is an important component that needs to be included to ensure that your project meets the necessary specifications :+1:.